vuu-ui/packages/vuu-shell/src/layout-management/SaveLayoutPanel.css

```diff
@@ -1,5 +1,122 @@
-.vuuSaveLayoutPanel {
-    background-color: aqua;
-    height: 400px;
-    width: 600px;
+* {
+  --salt-selectable-foreground-hover: #6d18bdc3;
+  --salt-selectable-foreground-selected: #6D18BD;
+  --salt-selectable-borderColor-selected: #6D18BD;
+  --saltInputLegacy-fontFamily: Nunito Sans;
+  --salt-text-fontFamily: Nunito Sans;
+  --saltInputLegacy-fontSize: 12px;
+  --salt-text-label-fontSize: 10px;
+  --saltFormFieldLegacy-label-paddingLeft: 0;
+  font-family: Nunito Sans;
+}
+
+.panelContainer {
+  display: flex;
+  padding: 16px;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+}
+
+.panelContent {
+  display: flex;
+  align-items: flex-start;
+  gap: 32px;
+}
+
+.formContainer {
+  display: flex;
+  width: 217px;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+}
+
+.settingsGroup {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  gap: 10px;
+  width: 100%;
+  line-height: 16px;
+}
+
+.setting {
+  display: flex;
+  height: 24px;
+  align-items: center;
+  gap: 6px;
+  color: var(--light-text-primary, #15171B);
+  font-feature-settings: 'ss02' on, 'ss01' on, 'salt' on, 'liga' off;
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.screenshotContainer {
+  display: flex;
+  width: 304px;
+  height: 208px;
+  padding: 11px 15.5px;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #E8E8E8;
+  background: var(--dark-text-primary, #FFF);
+}
+
+.screenshot {
+  /* Uncomment once screenshot functionality is implemented */
+  /* background: url(<path-to-image>), lightgray 50% / cover no-repeat; */
+  width: 273px;
+  height: 186px;
+  flex-shrink: 0;
+}
+
+.buttonsContainer {
+  display: flex;
+  padding-top: 8px;
+  justify-content: flex-end;
+  align-items: flex-start;
+  gap: 8px;
+  align-self: stretch;
+}
+
+.cancelButton,
+.saveButton {
+  display: flex;
+  height: fit-content;
+  padding: 4px 8px;
+  align-items: flex-start;
+  gap: 8px;
+  border-radius: 6px;
+  font-feature-settings: 'ss02' on, 'ss01' on, 'salt' on, 'liga' off;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 133.333%;
+  letter-spacing: 0.48px;
+  text-transform: uppercase;
+}
+
+.cancelButton {
+  color: var(--actionable-primary-foreground-default, #606477);
+  background: var(--actionable-primary-background-default, #FFF);
+}
+
+.saveButton {
```

Comment:
class names like this maybe a bit too generic, I tend to prefix them with component name, although they likely won't need individual styling like this once all the theme styles are in place.